docs/source/srs.rst

```diff
@@ -0,0 +1,283 @@
+  need to refer to this document throughout the development process to ensure
+  all target needs are met within the agreed upon constraints.
+* Integration Team Members: RIT EVT team members who handle systems level
+  integration on the team. These team members may use this document to gain
+  and understanding of how the DEV1 IMU will operate. Most critically, how the
+  DEV1 IMU will operate within the structure of the DEV1 architecture.
+
+The document is laid out where it is not strictly necessary to read all sections
+in their entirety. Instead each section should be self-contained and 
+comprehensive within its scope. As such audience members are encouraged to read 
+the sections that most pertain to their needs. Additionally, the glossary in the
+Appendix may prove useful in situations where phrases may be used in specific 
+ways within this document.
+
+Product Scope
+-------------
+
+The DEV1 IMU measures acceleration and position data for the DEV1 bike. The
+DEV IMU reports its measurements over the DEV1 CAN network to provide a
+general overview of the bike's orientation for other boards to act
+accordingly for optimal performance.
+
+Overview
+--------
+
+The rest of this software design document will go further into the specifics of 
+the requirements while also looking at the constraints of the system. The goal 
+is to clarify the use cases of the DEV1 IMU and to specify what the DEV1 IMU 
+will do in those cases. Design considerations will not be discussed, however 
+notable design constraints will be covered.
+
+
+Overall Description
+===================
+
+Product Perspective
+-------------------
+
+The DEV1 IMU plays a role in performance optimization for the DEV1 bike. It
+reports measurements such as linear/angular acceleration and Euler over the
+bike's CAN network, allowing other boards to adjust their configurations to
+maintain the best performance.
+
+The DEV1 IMU is made up of two components, the STM32F334K8 microcontroller
+and the BNO055 orientation sensor chip. The ST microcontroller handles
+processing of data and manages CAN network communication for the IMU. The
+BNO055 is used to obtain the orientation measurements.
+
+* list some boards that will use the data
+* difference between linear acceleration data and accelerometer data?
+
+User Interfaces
+~~~~~~~~~~~~~~~
+
+Users will rarely interact directly with the DEV1 IMU software. The DEV1 IMU 
+software will mainly be interfaced with via CANopen and thus will require 
+additional tools to interact with the DEV1 IMU. There is no current plan 
+for a team-developed tool to provide an interface for interaction with the 
+DEV1 IMU.
+
+Hardware Interfaces
+~~~~~~~~~~~~~~~~~~~
+
+The DEV1 IMU is connected to the bike's CANopen network which is made up of a
+two-wire differential pair. The on-board ST microcontroller is programmable
+through JTAG.
+
+Software Interfaces
+~~~~~~~~~~~~~~~~~~~
+
+The DEV1 IMU software interface is the report of measurements over the CAN
```

Comment:
this is a weird way to phrase this. Maybe say something like "The DEV1 IMU software reports measurements over the CAN (is it CAN or CANopen?) network.